TikTokApi/api/user.py

```diff
@@ -194,6 +194,65 @@ async def liked(
                 return
 
             cursor = resp.get("cursor")
+    
+    async def sounds(self, count=30, cursor=0, **kwargs) -> Iterator[dict]:
+        """
+        Returns a user's sounds.
+
+        Args:
+            count (int): The amount of sounds you want returned.
+            cursor (int): The offset of sounds from 0 you want to get.
+
+        Yields:
+            dict: A dictionary containing information about each sound.
+
+        Raises:
+            InvalidResponseException: If TikTok returns an invalid response.
+
+        Example Usage:
+            async for sound in api.user(username="artist_name").sounds():
+                # do something with each sound
+        """
+        sec_uid = getattr(self, "sec_uid", None)
+        if sec_uid is None or sec_uid == "":
+            await self.info(**kwargs)
+
+        user_id = getattr(self, "user_id", None)
+        if user_id is None or user_id == "":
+            raise ValueError("User ID is required to fetch sounds.")
```

Comment:
Nit: `not '' == True`
```suggestion
        if not sec_uid:
            await self.info(**kwargs)

        user_id = getattr(self, "user_id", None)
        if not user_id:
            raise ValueError("User ID is required to fetch sounds.")
```